.github/workflows/flaky.yaml

```
@@ -87,7 +87,7 @@ jobs:
             echo "$failure" >> $GITHUB_OUTPUT
           done
           echo "" >> $GITHUB_OUTPUT
-          echo "See https://github.com/n0-computer/iroh/actions/workflows/flaky.yaml" >> $GITHUB_OUTPUT
+          echo "See https://github.com/n0-computer/iroh-gossip/actions/workflows/flaky.yaml" >> $GITHUB_OUTPUT
```

```suggestion
          echo "See https://github.com/${{ github.repository }}/actions/workflows/flaky.yaml" >> $GITHUB_OUTPUT
```

Let's avoid future copy-paste errors like this